bitnami/mongodb/templates/network-policy.yaml

```
@@ -0,0 +1,46 @@
+{{- /*
+Copyright VMware, Inc.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if and .Values.networkPolicy.enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ include "mongodb.fullname" . }}
+  namespace: {{ include "mongodb.namespace" . | quote }}
```

Are these changes based on some other existing chart? Did you check any of the existing `networkpolicy.yaml` templates in particular? I would suggest that you check a recent one: bitnami/mlflow/templates/tracking/networkpolicy.yaml that should include all the best practices. Just as an example:

```suggestion
  name: {{ template "mongodb.fullname" . }}
  namespace: {{ include "common.names.namespace" . | quote }}
```

---

Yea i was checking out the postgres and redis `networkpolicy.yaml` templates which used .Release.Namespace and some other templates in mongoDB used the mongodb.namespace syntax so i went with it to have the consistency with mongo chart. However I can change it to the proposed one